routes/cards.js

```diff
@@ -0,0 +1,199 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+
+const { Card } = require('../models/card');
+const { User } = require('../models/user');
+
+function validateAmbassador(userId) {
+  if (!userId) {
+    return Promise.resolve();
+  }
+  return User.findOne({ _id: userId })
+    .then(result => {
+      if (!result) {
+        return Promise.reject('InvalidUser');
+      } else if (!result.ambassador) {
+        return Promise.reject('NotAmbassador');
+      }
+    });
+}
+
+/* ========== GET/READ ALL ITEM ========== */
+router.get('/cards', (req, res, next) => {
```

Comment:
Could we add a way to get a subset of cards from this as well?

Reply:
I'm not sure what you mean by a subset of cards. Could you explain what you mean?